CONTRIBUTING.md

```diff
@@ -78,3 +78,11 @@ If your pull request fixes an issue, add a comment with the word "Fixes", "Resol
 >   Fixes #1145
 
 If you need to, feel free to add comments to the PR asking for suggestions or help.
+
+### Bash scripts
+
+### Other scripts
+
+Bash scripts are used for more complex tasks such as updating TZDB. Many contributors will never need to run these scripts as they're, but just occasionally they may be useful when investigating a CI failure.
```

Comment:
Pretty sure there's a typo here: "as they're," - though I'm not sure sure what it should say.

Reply:
Whoops, thanks. Will fix.